rubyforgood/human-essentials
# 3340 add all settings org view

Resolves #3340 

### Description
This pull request solves the issue of not all of the editable fields for a bank being visible on the view page for the organization. I added the following fields:

- URL
<img width="241" alt="image" src="https://user-images.githubusercontent.com/65237366/215784824-503e588a-c797-402a-91a9-9b2a2b0566ee.png">
- Short Name
<img width="256" alt="image" src="https://user-images.githubusercontent.com/65237366/215784891-bafb8b6a-b127-482f-a186-337ae857b94e.png">
- Required Partner Fields
<img width="340" alt="image" src="https://user-images.githubusercontent.com/65237366/215784960-61dc2d74-e4ce-4ca1-9f87-379d32b32106.png">
- Child based, individual, and quantity based requests
<img width="361" alt="image" src="https://user-images.githubusercontent.com/65237366/215785071-f9f9a917-274a-4ebd-b1b6-df96c15cd476.png">
- Custom partner invitation message
<img width="464" alt="image" src="https://user-images.githubusercontent.com/65237366/215785173-e4b05401-3433-4442-a5c6-4e5e719fd8d3.png">
- Logo
<img width="550" alt="image" src="https://user-images.githubusercontent.com/65237366/215785229-d65afb49-46af-4851-b6d7-67938d8edd6b.png">

I added tests in `spec/system/organization_system_spec.rb` that look for this new content on that page. I also updated the readme to include instructions on how to run not just the whole test suite, but a single test spec. There were some fields which were not displaying text "Not defined/provided" when the organization attributes in question were indeed undefined, so I fixed those too.

### Type of change

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)
* Documentation update

### How Has This Been Tested?



- Log in as an organization admin
- Click on the `My Organization` tab in the left hand side bar
- At the bottom of the `Organization Info` card, click the `edit` button.
- Ensure all of the fields mentioned above have a value (LOGO SPECIFICALLY WILL _NOT_ BE DISPLAYED ON THE VIEW PAGE UNLESS A LOGO IS UPLOADED)
- Click the save button when you are finished making updates
- Ensure that all of the fields mentioned above do indeed exist, and that everything looks correct.
